tdvt/tdvt/config_gen/gentests.py

```diff
@@ -188,124 +204,46 @@ def list_configs(ds_registry, target_config_name=None):
-    staples_fields.append('[Order Quantity]'           )
-    staples_fields.append('[Sales Total]'              )
-    staples_fields.append('[Discount]'                 )
-    staples_fields.append('[Tax Rate]'                 )
-    staples_fields.append('[Ship Mode]'                )
-    staples_fields.append('[Fill Time]'                )
-    staples_fields.append('[Gross Profit]'             )
-    staples_fields.append('[Price]'                    )
-    staples_fields.append('[Ship Handle Cost]'         )
-    staples_fields.append('[Employee Name]'            )
-    staples_fields.append('[Employee Dept]'            )
-    staples_fields.append('[Manager Name]'             )
-    staples_fields.append('[Employee Yrs Exp]'         )
-    staples_fields.append('[Employee Salary]'          )
-    staples_fields.append('[Customer Name]'            )
-    staples_fields.append('[Customer State]'           )
-    staples_fields.append('[Call Center Region]'       )
-    staples_fields.append('[Customer Balance]'         )
-    staples_fields.append('[Customer Segment]'         )
-    staples_fields.append('[Prod Type1]'               )
-    staples_fields.append('[Prod Type2]'               )
-    staples_fields.append('[Prod Type3]'               )
-    staples_fields.append('[Prod Type4]'               )
-    staples_fields.append('[Product Name]'             )
-    staples_fields.append('[Product Container]'        )
-    staples_fields.append('[Ship Promo]'               )
-    staples_fields.append('[Supplier Name]'            )
-    staples_fields.append('[Supplier Balance]'         )
-    staples_fields.append('[Supplier Region]'          )
-    staples_fields.append('[Supplier State]'           )
-    staples_fields.append('[Order ID]'                 )
-    staples_fields.append('[Order Year]'               )
-    staples_fields.append('[Order Month]'              )
-    staples_fields.append('[Order Day]'                )
-    staples_fields.append('[Order Date]'               )
-    staples_fields.append('[Order Quarter]'            )
-    staples_fields.append('[Product Base Margin]'      )
-    staples_fields.append('[Product ID]'               )
-    staples_fields.append('[Receive Time]'             )
-    staples_fields.append('[Received Date]'            )
-    staples_fields.append('[Ship Date]'                )
-    staples_fields.append('[Ship Charge]'              )
-    staples_fields.append('[Total Cycle Time]'         )
-    staples_fields.append('[Product In Stock]'         )
-    staples_fields.append('[PID]'                      )
-    staples_fields.append('[Market Segment]'           )
-
-    #Go through input and top level subdirs. Create those in the output and then process the files.
+    # TODO: Logic will go here to look at ds_registry and either use calcs/staples or use custom table;
+    #       that table or those tables will be used in process_test_files below.
+
```

Comment:
These are moved to `constants.py`. This was originally written with closing `)` aligned; looks nice but doesn't affect anything.